File: python/0119.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        a=[1]
        s=1
        for i in range(1,rowIndex+1):
            s=s*(1+rowIndex-i)/i
            a.append(s)
        print(a)
        return a

File: rust/src/bin/0119.rs
struct Solution;

impl Solution {
    pub fn get_row(row_index: i32) -> Vec<i32> {
        if row_index == 0 {
            return vec!(1);
        }
        let mut a = vec!(1);
        for i in 1..=row_index as usize {
            let now = a[i - 1] as i64 * (row_index as i64 + 1 - i as i64) / i as i64;
            a.push(now as i32);
        }
        a
    }
}

fn main() {
    println!("{:?}", Solution::get_row(4));
}